src/test/java/Homework19.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        String notification = "Deleted playlist \"firstplaylist.\"";
+
+        provideEmail("lolitamantsiuk@gmail.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        choosePlaylist();
+        XPlaylist();
+//        confirmDelete();
+        Assert.assertEquals(notificationPlaylistDeleted(), notification);
+    }
+
+    public void choosePlaylist() throws InterruptedException {
+        WebElement playlist = driver.findElement(By.xpath("//section[@id='playlists']//a[@href='#!/playlist/74360']"));
```

Comment:
Hello, 
Your homework 19 has been reviewed, your have used `@Parameters` for passing baseUrl from the TestNG config file to the tests. Great!
But it seem like test script Homework19 run not stable. You defined playlist selector use href attribute with id of playlist. I recommend that you use the text playlist name here to click and confirm the notification delete sucess playlist

//HINT:
![image](https://github.com/sofia-testpro/A51-QA-Automation-Framework/assets/144403954/1b84bf2d-17c8-4ad5-8a02-0d3ad835b0eb)